kubernetes-extension-examples/examples/kubernetes-deployment/kubernetes_deployment.out

```diff
@@ -27,15 +27,15 @@ Generating artifacts...
 	kubectl apply -f ./kubernetes
 
 	Run the following command to install the application using Helm: 
```

Comment:
```suggestion
	Execute the below command to install the application using Helm: 
```

Reply:
These are outputs of the build command with annotations.